corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -263,7 +263,7 @@
                 <div class="accordion-body collapse 
                     {% if cases|length == 1 %}in{% endif %}" 
                     id="form-case-acc-{{ forloop.counter }}">
-                    <div class="accordion-inner">
+                    <div class="accordion-inner" style="display: inline-block">
```

Comment:
I can't find the reason it broke, either, but adding an inline style here isn't great. Looking at the page, the content is spilling because the `property-table-container` is the only child of `property-table-row` and is floated left, so there's no non-floating content to give it height. 

It's likely safe to remove the float rule from `property-table-container`, though you might want to check that it displays right for both property_table.html and dl_property_table.html. Alternatively, you can add clearfix to property-table-container (should be safe to do this in proptable/style.css, rather than adding it as an extra class in the html files).